src/graph/impl/KokkosGraph_Distance2Color_impl.hpp

```diff
@@ -565,64 +559,73 @@ class GraphColorDistance2
       color_type colorBase;
       forbidden_view forbidden;         //forbidden color bitset for columns
       color_view_type colors;
-      const_lno_row_view_type Crowmap;  //C <-> V graph (row c is the vertices incident to c)
-      const_lno_nnz_view_t    Ccolinds;
-      const nnz_lno_type numVerts;
+      rowmap_t Crowmap;  //C <-> V graph (row c is the vertices incident to c)
+      entries_t Ccolinds;
+      const lno_t numVerts;
     };
 
     template<int batch>
```

Comment:
Does this have to be template parameter ?

Reply:
@srajama1 I was worried about efficiency with doing it at runtime, although I really need to measure it. Batch is always a power of 2 so it could still be represented by bitshift. I do see a code size concern with compile time.

Reply:
I'm leaving batch as a template parameter for now but I can revisit this after the exawind code freeze.